app/controllers/needs_controller.rb

```diff
@@ -192,4 +192,13 @@ def assigned_to_me(assigned_to)
     assigned_to = "user-#{current_user.id}" if assigned_to == 'assigned-to-me'
     assigned_to
   end
+
+  def construct_assigned_to_options
```

Comment:
I think this is a merge issue, there is already another `construct_assigned_to_options` in this controller, outside of the private methods

Reply:
Sure, working on it.

Reply:
When I try to start the app. Maybe overlaps with Riccardo's comment

![image](https://user-images.githubusercontent.com/63452375/81831457-0dccdb00-9546-11ea-9105-a33129ae0957.png)

Reply:
Yes, I tried to resolve the conflicts using Git web and did mistake. Working to fix it.

Reply:
Select Box is to small, And didn't we have an avatar in the designs?

![image](https://user-images.githubusercontent.com/63452375/81834818-2a6b1200-954a-11ea-9623-9594c4843807.png)

Reply:
This is not what I did with the styling. Strange where the styles lost.

Reply:
And also, the buttons are on the right hand side.

And we also have a checkbox for sending emails. Maybe we decided to leave it out?

Reply:
About the avatar I dont set avatar, because we dont have avatar for different roles and it will always be the same.

Reply:
> And also, the buttons are on the right hand side.
> 
> And we also have a checkbox for sending emails. Maybe we decided to leave it out?

I think email sending is part of something that @jamiebuckley already did. Not sure how to connect it here.